Source/Timeline/EventView.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+import DateTools
+import Neon
+import DynamicColor
+
+class EventView: UIView {
```

Comment:
I guess it's good to declare it public.
I am not sure what layer will actually ask for data, either `DayView` or some intermediary which will create (reuse) `EventViews` form models for DayView.

Declaring it public will ease customization